grace/base.py

```diff
@@ -34,6 +34,15 @@ class Annotation(enum.IntEnum):
     UNKNOWN = 2
 
 
+class Prediction(enum.Enum):
+    """Values of predicted label & logits."""
+
+    LABEL: int = "predicted class label (argmax)"
+    PROB_TN: float = "probability of true negative detection (softmax)"
+    PROB_TP: float = "probability of true positive detection (softmax)"
+    # PROB_UNKNOWN: float = "probability of unknown detection (softmax)"
```

Comment:
I don't have time to review this, but this doesn't look correct to me. It shouldn't be an enum and the typing is wrong given the values. Aso, the logits are unscaled, so perhaps change the docstring

Reply:
I think it should just be a regular class, or a dataclass

Reply:
I don't think I fully understand what's the difference between this and the `GraphAttrs` class as defined above, which also takes in a mixture of datatypes, but I'm following the suggestion & implementing the change now.